Makefile

```diff
@@ -191,13 +191,17 @@ mod-tidy:
 	go mod download
 	go mod tidy
 
+check-mod-tidy: # @HELP Check that mod files are tidy.
+	go mod tidy -diff
+	cd hack/pulumi && go mod tidy -diff
+
```

Comment:
At some point these rules/targets got sorted alphabetically, I'd rename it to `mod-tidy-check` to keep them together.